packages/extension/src/AIPromptDialog/AIPromptDialog.tsx

```diff
@@ -130,6 +246,16 @@ const AIPromptDialogContent: React.SFC<any> = (props: AIPromptDialogProps) => {
         title={strings.title}
       ></DialogHeader>
       <DialogContent>
+        {keywords ? (
```

Comment:
Should this be `keywords.length`? It looks like it will default to an empty array and this would still show the keywords component.